classes/fullstack/project1/server/src/auth/index.ts

```diff
@@ -0,0 +1,25 @@
+import { hashPassword, verifyPassword } from '../utils/bcrypt'
```

Comment:
this is not part of your issue scope please remove this file

Reply:
ok. removed